ibtk/include/ibtk/FEDataInterpolation.h

```diff
@@ -235,7 +243,7 @@ class FEDataInterpolation
     size_t registerInterpolatedSystem(const libMesh::System& system,
                                       const std::vector<int>& vars = std::vector<int>(1, 0),
                                       const std::vector<int>& grad_vars = std::vector<int>(),
-                                      libMesh::NumericVector<double>* system_data = nullptr);
+                                      libMesh::NumericVector<double>* system_vec = nullptr);
```

Comment:
There is a class named `SystemData` so I thought I would try to make it clearer that this is the vector of DOFs.